testing/rspamd/APKBUILD

```diff
@@ -35,6 +35,7 @@ build() {
 		-DRSPAMD_USER=$pkgusers \
 		-DRSPAMD_GROUP=$pkggroups \
 		-DENABLE_HIREDIS=ON \
+		-DENABLE_LUAJIT=OFF \
```

Comment:
Hm, it seems that @ncopa didn’t want to disable PaX memory protection for rspamd for some reason…?